x-pack/plugins/security_solution/public/management/pages/policy/view/policy_details.test.tsx

```diff
@@ -215,60 +221,118 @@ describe('Policy Details', () => {
-      });
+        it('should not display the host isolation exceptions tab with no privileges and no assigned exceptions', async () => {
+          findExceptionsApiHttpMock.responseProvider.exceptionsFind.mockReturnValue({
+            data: [],
+            total: 0,
+            page: 1,
+            per_page: 100,
+          });
+          policyView = render();
+          await asyncActions;
+          policyView.update();
+          await waitFor(() => {
+            expect(findExceptionsApiHttpMock.responseProvider.exceptionsFind).toHaveBeenCalled();
+          });
+          expect(policyView.find('button#hostIsolationExceptions')).toHaveLength(0);
+        });
 
-      it('should not display the host isolation exceptions tab with no privileges and no assigned exceptions', async () => {
-        findExceptionsApiHttpMock.responseProvider.exceptionsFind.mockReturnValue({
-          data: [],
-          total: 0,
-          page: 1,
-          per_page: 100,
+        it('should not display the host isolation exceptions tab with no privileges and no data', async () => {
+          findExceptionsApiHttpMock.responseProvider.exceptionsFind.mockReturnValue({
+            data: [],
+            total: 0,
+            page: 1,
+            per_page: 100,
+          });
+          policyView = render();
+          await asyncActions;
+          policyView.update();
+          await waitFor(() => {
+            expect(findExceptionsApiHttpMock.responseProvider.exceptionsFind).toHaveBeenCalled();
+          });
+          expect(policyView.find('button#hostIsolationExceptions')).toHaveLength(0);
         });
-        policyView = render();
-        await asyncActions;
-        policyView.update();
-        await waitFor(() => {
-          expect(findExceptionsApiHttpMock.responseProvider.exceptionsFind).toHaveBeenCalled();
+
+        it('should display the host isolation exceptions tab with no privileges if there are assigned exceptions', async () => {
+          policyView = render();
+          await asyncActions;
+          policyView.update();
+          await waitFor(() => {
+            expect(findExceptionsApiHttpMock.responseProvider.exceptionsFind).toHaveBeenCalled();
+          });
+          expect(policyView.find('button#hostIsolationExceptions')).toHaveLength(1);
         });
-        expect(policyView.find('button#hostIsolationExceptions')).toHaveLength(0);
       });
 
-      it('should not display the host isolation exceptions tab with no privileges and no data', async () => {
-        findExceptionsApiHttpMock.responseProvider.exceptionsFind.mockReturnValue({
-          data: [],
-          total: 0,
-          page: 1,
-          per_page: 100,
-        });
-        policyView = render();
-        await asyncActions;
-        policyView.update();
-        await waitFor(() => {
-          expect(findExceptionsApiHttpMock.responseProvider.exceptionsFind).toHaveBeenCalled();
+      describe('Trusted Apps tab', () => {
```

Comment:
Idea: when going through all the other tabs, this test group could be transformed to a `describe.each`.